packages/pn-personafisica-webapp/src/redux/notification/__test__/reducers.test.ts

```diff
@@ -98,6 +99,7 @@ describe('Notification detail redux state tests', () => {
 
   afterEach(() => {
     mock.reset();
+    sessionStorage.removeItem(PAYMENT_CACHE_KEY);
```

Comment:
Maybe now it is useless. Please do a check

Reply:
I need that only before "Should be able to fetch payment info and not replace the payment if is equal"